File: java/program68.java
import java.util.*;

class program68
{
    public static void DisplayCapitalLetters1(String str)
    {
        char Arr[] = str.toCharArray();
        int j = 0;
        int count = 0;

        for(int i = 0; i < Arr.length; i++)
        {
            if((Arr[i] >= 'A') && (Arr[i] <= 'Z'))
            {
                count++;
            }
        }

        char Brr[] = new char[count];

        for(int i = 0; i < Arr.length; i++)
        {
            if((Arr[i] >= 'A') && (Arr[i] <= 'Z'))
            {
                Brr[j] = Arr[i];
                j++;
            }
        }

        String str1 = new String(Brr);
        System.out.println("Editing string is : "+str1);
    }

    public static void DisplayCapitalLetters(String str)
    {
        char Arr[] = str.toCharArray();
        char Brr[] = new char[Arr.length];
        int j = 0;

        for(int i = 0; i < Arr.length; i++)
        {
            if((Arr[i] >= 'A') && (Arr[i] <= 'Z'))
            {
                Brr[j] = Arr[i];
                j++;
            }
        }
        String str1 = new String(Brr);
        System.out.println("Editing string is : "+str1);
    }

    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter String");
        String str = sobj.nextLine();

        DisplayCapitalLetters1(str);
    }
}

File: cpp/program68.cpp
// program185.c

#include<iostream>
using namespace std;

void strcmpcapX(char *str,char *dest)
{
    while(*str != '\0')
    {
        if((*str >= 'A') && (*str <= 'Z'))
        {
            *dest = *str;
            dest++;
        }
        str++;
    }
    *dest = '\0';
}

int main()
{
    char Arr[30];
    char Brr[30];

    cout<<"Enter string \n";
    cin.getline(Arr,50);

    strcmpcapX(Arr,Brr);

    cout<<"String after editing is : "<<Brr<<"\n";

    return 0;
}